x/nexus/exported/types.go

```diff
@@ -336,3 +336,16 @@ func (m GeneralMessage) Type() MessageType {
 
 	return TypeGeneralMessageWithToken
 }
+
+// FromGeneralMessage returns a WasmMessage from a GeneralMessage
+func FromGeneralMessage(msg GeneralMessage) WasmMessage {
```

Comment:
```suggestion
func MakeWasmMessage(msg GeneralMessage) WasmMessage {
```

I feel like using `From` here is weird because there is no receiver.

Reply:
but `::from` in rust has no "receiver" either.

Reply:
It sort of does because from is a trait implemented for a particular type. Versus this is more of just a free floating function. What if we want to convert a general message to a different type? In Rust we implement from for each of those types, but here we can't.

Reply:
It's the same argument with `MakeWasmMessage`. What if we want to convert another type to a WasmMesssage"?